test/multiline-invalid.md

```diff
@@ -4,8 +4,8 @@
 <div something="row">
     <button type="invalid">
         Button not correctly closed
-    </button
```

Comment:
Parser errors have changed so snapshots failed on all versions, this change works for all versions.